src/Events/Calendar_Embeds/Calendar_Embeds.php

```diff
@@ -0,0 +1,168 @@
+<?php
+/**
+ * Manages the External Calendar Embeds Feature.
+ *
+ * @since TBD
+ *
+ * @package TEC\Events\Calendar_Embeds
+ */
+
+namespace TEC\Events\Calendar_Embeds;
+
+use Tribe__Events__Main as TEC;
+
+/**
+ * Class Calendar_Embeds
+ *
+ * @since TBD
+ *
+ * @package TEC\Events\Calendar_Embeds
+ */
+class Calendar_Embeds {
+
+	/**
+	 * Calendar Embeds post type slug.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	const POSTTYPE = 'tec_calendar_embed';
+
+	/**
+	 * The page slug.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public static $slug = 'calendar-embeds';
```

Comment:
I would avoid this being static or public, expose a `get_url()` method that has access to a protected prop.